cloud/general/init-ffmpeg.sh

```diff
@@ -42,13 +42,17 @@ cd $_ASGS_TMP
 if [ ! -e ${NASM_TGZ} ]; then
   wget --no-check-certificate https://www.nasm.us/pub/nasm/releasebuilds/${NASM_VERSION}/${NASM_TGZ}
 fi
-
-rm -rf $NASM_DIR 2> /dev/null
-tar zxvf $NASM_TGZ
-cd $NASM_DIR
-./configure --prefix=$OPT
-make
-make install
+# check that there are no issues with downloading nasm
+if [[ -e ${NASM_TGZ} && -s ${NASM_TGZ} &&  "$(file nasm-2.15.05.tar.gz)" =~ "gzip compressed data" ]]; then
```

Comment:
```suggestion
if [[ -e ${NASM_TGZ} && -s ${NASM_TGZ} &&  "$(file ${NASM_TGZ} )" =~ "gzip compressed data" ]]; then
```

Reply:
Did you mean this? Also, checkout we can suggest smallish fixes in the code review itselft, neat!

(press this button to do so, ![image](https://user-images.githubusercontent.com/46434714/210699382-fc7300fa-66e7-4d96-b613-1c38d599b19b.png))